Original patch by Kevin DeKorte <kdekorte at gmail dot com>
Recreated by Phillip Warner <pc_warner@yahoo.com>

--- src/gui.c.orig	2009-10-16 19:46:09.000000000 -0500
+++ src/gui.c	2009-10-16 19:47:05.000000000 -0500
@@ -94,7 +94,9 @@
      * onwards, requesting the native window in a thread causes a BadWindowID,
      * so we need to request it now. We could call gdk_window_ensure_native(),
      * but that would mean we require GTK+ 2.18, so instead we call this */
+#ifdef GTK2_14_ENABLED
     GDK_WINDOW_XID(gtk_widget_get_window(GTK_WIDGET(widget)));
+#endif
 }
 
 gint get_player_window()
